packages/manager/apps/web/client/app/private-database/order/clouddb/translations/Messages_fr_FR.json

```diff
@@ -33,5 +33,8 @@
   "private_database_order_clouddb_payment_checkout_success": "Votre commande pour votre base de données Web Cloud Databases a bien été prise en compte. Votre compte {{accountId}} a bien été débité de {{price}}.",
   "private_database_order_clouddb_payment_checkout_success_link": "Voir ma facture",
   "private_database_order_clouddb_bill_success": "Votre {{link}} a été généré avec succès. Votre base de données Web Cloud Databases sera active après le paiement de celui-ci.",
-  "private_database_order_clouddb_bill_success_link": "bon de commande"
+  "private_database_order_clouddb_bill_success_link": "bon de commande",
+
+  "ovhManagerHostingDatabaseOrderPublic_success_noCheckout": "Votre commande est prise en compte. Votre produit sera activé dans quelques instants.",
+  "ovhManagerHostingDatabaseOrderPublic_success_checkout": "Votre <a href=\"{{url}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé."
```

Comment:
```suggestion
  "private_database_order_clouddb_bill_success_noCheckout": "Votre commande est prise en compte. Votre produit sera activé dans quelques instants.",
  "private_database_order_clouddb_bill_success_checkout": "Votre <a href=\"{{url}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé."
```
And remove these no used anymore

Reply:
These are used otherwise we won't have the following banneers:

![image](https://github.com/ovh/manager/assets/8767496/b5ec4aa9-42e3-4388-b010-9d3166296b9c)

But I can rename it accordingly yes